hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -835,8 +844,15 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 	if err != nil {
 		return ctrl.Result{}, fmt.Errorf("failed to hash HostedCluster configuration: %w", err)
 	}
+	additionalTrustBundle := ""
+	if hcluster.Spec.AdditionalTrustBundle != nil {
+		additionalTrustBundle, err = r.getAdditionalTrustBundle(ctx, hcluster)
+	}
+	if err != nil {
```

Comment:
Shouldn't this just go inside the if block and be with the err assignment on L849?